### PR TITLE
Soft Descriptor and CrossCurrency Features

### PR DESCRIPTION
**Pull request by waskalevikas**
Soft Descriptor and CrossCurrency new Features
URL Updates 
Long term timeout fix

### CODE REVIEW

**Review by sweeneyjay — DISMISSED** [review]

File: config/tenant.json
[the remark this answers is not in the document]
Comment (waskalevikas) [reply config/tenant.json:1]: Didn't understood, can you elaborate pls? Also, we didn't changed this file.

File: docs/documentation/features/softdescriptor.md
[the remark this answers is not in the document]
Comment (pkrishk) [reply docs/documentation/features/softdescriptor.md:1]: We are sending "statementDescriptor" in the request payload. DDP internally sends this value as "statement_descriptor" to Mastercard API.

[the remark this answers is not in the document]
Comment (sweeneyjay) [reply docs/documentation/features/softdescriptor.md:1]: Remember this is client facing documentation and we should be careful not to reveal internal workings of our code or mappings when possible. I think in this case the point we are trying to make is that:
A) There is a descriptor in the request - when send that to Visa/Mastercard
B) If there is no descriptor in the request - We send what is configured during onboarding

Let's reword this to follow that Idea. We can can give the specific key name in the DDP request but we don't need to mention the key name that is sent to downstream.

File: config/tenant.json
[the remark this answers is not in the document]
Comment (sweeneyjay) [reply config/tenant.json:1]: @vikaswaskale in this file there is a default Value for the key "longDescription" - This should be a description that refers to DDP. Currently it is displaying a description for DevStudio the software.

**Review by vmaridu-fiserv — DISMISSED** [review]
